Source/Scene/Camera.js

```diff
@@ -2376,7 +2376,7 @@ define([
      * @param {Object} [options.orientation] An object that contains either direction and up properties or heading, pith and roll properties. By default, the direction will point
      * towards the center of the frame in 3D and in the negative z direction in Columbus view or 2D. The up direction will point towards local north in 3D and in the positive
      * y direction in Columbus view or 2D.
-     * @param {Number} [options.duration=3.0] The duration of the flight in seconds.
+     * @param {Number} [options.duration=3.0] The max duration of the flight in seconds.
```

Comment:
Thanks @klim705 but this is not actually correct.  If you specify a duration the flight always lasts that exact amount of time, it's only when you don't specify a value that the time varies.  Let's change it to something like: 

_The duration of the flight in seconds. If omitted, Cesium attempts to calculate an ideal duration based on the distance to be traveled by the flight._

I would also get rid of the `=3.0` default value since it's incorrect since #2825.  We should probably eventually add a `maxDuration` property to make this less confusing, but let's just leave it as is for now and improve the doc.